code/controllers/subsystem/pathfinder.dm

```diff
@@ -0,0 +1,46 @@
+SUBSYSTEM_DEF(pathfinder)
+	name = "Pathfinder"
+	init_order = INIT_ORDER_PATH
+	flags = SS_NO_FIRE
+	var/datum/flowcache/mobs
+	var/static/space_type_cache
+
+/datum/controller/subsystem/pathfinder/Initialize()
+	space_type_cache = typecacheof(/turf/space)
+	mobs = new(10)
+	return ..()
```

Comment:
This being an SS not a global datum kinda pains me given it does no work and isn't init order critical.

However, thats not going to block this PR and is merely just a comment. I firmly believe any SS that doesnt require critical load order and doesnt fire should just be a misc global datum because god damn we have too many subsystems.

Take the cameranet manager for an example. Thats its own thing for good reason. SSradio however needs to die. 

---

Again, not review blocking, just me having wishful thoughts.